src/views/Workspace/Cards/AddFindPlayground.jsx

```diff
@@ -0,0 +1,394 @@
+                        : `< ${Math.round(distance)} m`;
+
+                      return (
+                        <div key={name} className={classes.result}>
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Naam</div>
+                                <div className={classes.resultSubtitle}>{name}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Afstand</div>
+                                <div className={classes.resultSubtitle}>{convertedDistance}</div>
+                            </div>
+
+                            <div className={classes.resultProperty}>
+                                <div className={classes.resultTitle}>Deelneemers</div>
+                                <div className={classes.resultSubtitle}>{volunteerCount}</div>
+                            </div>
+
+                            <Button
+                              className={classes.resultButton}
+                              color="primary"
+                              component={Link}
+                              to={`/workspace/${id}`}
+                              variant="contained"
+                            >Sluit je aan</Button>
+                        </div>
+                      );
+                  })
+              }
+
+              <Button
+                className={classes.addPlaygroundButton}
+                onClick={this.toggleAddPlayground}
+              >
+                  Mijn speeltuin staat er nog niet bij
+                  <ArrowIcon />
+              </Button>
+          </div>
+        );
+    }
+
+    render() {
+        const { classes, googleMapsKey } = this.props;
+        const { isAddPlaygroundOpen, userAddress } = this.state;
+
+        return (
+          <div>
+              <WorkspaceCard
+                title={"Speeltuin toevoegen"}
+                done={null}
+                image={require("../../../assets/img/backgrounds/workspace-welcome.jpg")}
+                content={"De eerste stap is om je aan te sluiten bij de actiepagina van jouw speeltuin, of om deze aan te maken. Vul je gegevens in om te beginnen."}
+                expandContent={
+                    <div className={classes.wrapper}>
+                        <div className={classes.search}>
+                            <PlacesWithStandaloneSearchBox
+                              getResults={this.getResults}
+                              classes={classes}
+                              googleMapsKey={googleMapsKey}
+                            />
+
+                            <Button
+                              className={classes.gotoMapButton}
+                              onClick={this.toggleAddPlayground}
+                            >
+                                Of zoek op de kaart
```

Comment:
this was not meant to add a playground; only for searching on map. A button can be below the map to add a playground,

Reply:
made issue for this; this maybe needs beta testing, on how people use this.